src/index.html

```diff
@@ -13,14 +13,14 @@
     <title>Product cards</title>
   </head>
   <body>
-    <div class="card" data-qa="card">
+    <article class="card" data-qa="card">
```

Comment:
I don't think that `article` the best tag for this block.

It's more about text articles, more you could read [there](https://developer.mozilla.org/ru/docs/Web/HTML/Element/article)

Reply:
I changed it to `article` because I suppose it would fit better for the layout_catalog. There I have a `section`  as a block of cards. Cards are `articles` that contain `p` tags